x-pack/metricbeat/module/cloudfoundry/container/main_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package container
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/elastic/beats/v7/libbeat/paths"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Same for this test, is anything missing? could you add a comment explaining it if not?

Reply:
Ok, I will add some comments about why `TestMain` is being used.

Reply:
I have removed these files, they are not needed now that cache is only used in `add_cloudfoundry_metadata`.